README.md

```diff
@@ -42,7 +34,7 @@ Pre-provisioning this box front-loads internet access requirements, flakiness, a
   - Default Vagrant SSH key
   - Configure SSH daemon (disable DNS and API authn for speed)
   - Set box build time (`/etc/vagrant_box_build_time`)
-- VirtualBox
+- VirtualBox and VMware
```

Comment:
Is this right? do all the points below also apply to VMWare?

Also - let's call it "VMWare Fusion".

Reply:
The generated image _should_ work with VMware Workstation and Fusion, although I've only tested the latter.

Reply:
You're right though and we should be clear that that's what we mean :)